chore: translate the README and SOLUTION to English (#1)
MeilleursAgents/frontend-technical-test: README.md

# Meilleurs Agents Frontend Technical Test

[![MeilleursAgents logo](images/logo-ma.png)](https://www.meilleursagents.com)

## Presentation

> This reposository contains that instruction for MeilleursAgents Frontend Technical Test.

Reviewer: la je questionne mon anglais : on dit `contains that instruction` ou `contains the instructions` ?

> This reposository contains that instruction for MeilleursAgents Frontend Technical Test.

Reviewer 2: ```suggestion
This repository contains instructions for the MeilleursAgents Frontend Technical Test.
```

> It's objective is to give us an overview of your practical technical knowledge through the realisation of an application that matches what we do as Frontend Engineer at Meilleurs Agents.

Reviewer: its ?

Reviewer 3: Also, i'd rather put "goal' instead of objective

## Objective

> The objective is to develop the client side code of an application allowing the visualisation of all the message sent to a realtor agency based on a given API.

Reviewer: the message`s`

> The objective is to develop the client side code of an application allowing the visualisation of all the message sent to a realtor agency based on a given API.

Reviewer 2: ```suggestion
The aim is to develop the client side code of an application allowing visualizing all the messages sent to a realtor agency based on a given API.
```

Author: Je vais prendre ta tournure, mais avec `visualising` vs `visualizing`.

> The messages can be of different types: email, text message, voicemail.

Reviewer 2: ```suggestion
Messages can be of different types: email, text message, voicemail.
```

> The app is destined to be used by the agency collaborators.

Reviewer 2: ```suggestion
The app will be used by the agency collaborators.
```

⚠️ CVs alone are not allowed to this email address. To apply, all our job offers are available on our [career page](https://carriere.meilleursagents.com/) or directly on [Welcome to the Jungle](https://www.welcometothejungle.com/fr/companies/meilleursagents/jobs)

## Completion time

The time you spend on this test is not limited. The idea is to take your time, respect the assignment, and send us the result when you are happy with it.

## Logical and technical functionnalities

### Required for the test presentation step

**Product functionnalities:**

- I can switch agencies via both the select and the URL and I see the messages list for the selected agency.
- I can scroll through the messages list on several pages regardless of the screen size.
- I can click on a message and open the message details.
- If the message wasn't read, the counter decrements

**Technical implementation:**

- Develop the small and large screen version
- Develop the agency switch mecanism via both a native HTML select and the URL (routing)
- Develop the agency view with the messages list

> - Develop the possibily to read a message through the URL (routing)

Reviewer: `possibility`

- Respect the routing structure `/realtors/:id` and `/realtors/:id/messages/:id`
- Handle the unread message counter (send a request to the API to make the value persist)

> - Handle the pagination (infinate scroll)

Reviewer: `infinite`
